src/Mapbender/ManagerBundle/Resources/public/components.js

```diff
@@ -22,6 +22,45 @@ $(function() {
         return false;
     });
 
+    $(document).on('click', ".sortByColumn", function(){
+        var $this = $(this);
+        var asc = true;
+
+        $this.css('text-decoration', 'underline');
+        $this.siblings().css('text-decoration', 'none');
+        $this.siblings().find('span').hide();
+
+        if($this.find('span').is(":hidden")){
+            $this.find('span').removeClass('fa-arrow-down').addClass('fa-arrow-up').show();
+        }else if($this.find('span').hasClass('fa-arrow-up')){
```

Comment:
And please run the auto-formatter once before creating a PR, on some places (like here) the space is missing after the brace. This avoids a line showing as modified later when an unrelated change is made.